cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -286,7 +306,6 @@ public KafkaBrokerConfigurationBuilder withListeners(
         ////////////////////
         // Shared configurations with values dependent on all listeners
         ////////////////////
-
```

Comment:
This is intentional here. Separates unrelated comments.

Reply:
Clumsy error. Will fix